### PR TITLE
Ec 316 : added contract examples

### PR DESCRIPTION
**Pull request by ugulavaGeorge**

### CODE REVIEW

File: docs/smartContractExamples/LetterOfCredit [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Аккредитив
+# Транзакция предоставляет MultiSignature
+
+contract (signature: MultiSig, transaction: Transaction) = {
+  def isValidSig(signature: MultiSig, message: Array[Byte], key: Array[Byte]): Bool = {
+    true in signature.map(lamb (x: Array[Byte]) = checkSig(x, message, key))
```
Comment (oskin1) [inline docs/smartContractExamples/LetterOfCredit]: You'd probably better use `allOf(signature.map(lamb (x: Array[Byte]) = checkSig(x, message, key)))`

File: docs/smartContractExamples/TokenEmission [hunk 2]
```diff
@@ -0,0 +1 @@
+# Выполняется транзакцией с пометокой о выпуске
```
Comment (oskin1) [inline docs/smartContractExamples/TokenEmission]: Do we really need files with only explanation comment inside?

File: docs/smartContractExamples/escrow_contract.pr [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Эскроу-счет хранит ценность до наступления определенных обстоятельств 
+# Для примера используем накопительный счет - прячем от себя коины с целью предотвратить паническую продажу
+# Условие - высота блокчейна превышает 500 и сигнатура принадлежит нам
+
+contract (signature: Signature25519, transaction: Transaction, state: State) = {
+	let mySignature = base58'GtBn7qJwK1v1EbB6CZdgmkcvt849VKVfWoJBMEWsvTew'
```
Comment (Bromel777) [inline docs/smartContractExamples/escrow_contract.pr]: Is it signature or public key?

File: src/test/scala/org/encryfoundation/prismlang/integration/CostEstimatorSpec.scala [hunk 4]
```diff
@@ -1,10 +1,10 @@
 package org.encryfoundation.prismlang.integration
 
-import org.encryfoundation.prismlang.compiler.{CompiledContract, PCompiler}
 import org.scalatest.PropSpec
 import org.scalatest.concurrent.TimeLimits
 import org.scalatest.time.SpanSugar._
-import scala.util.{Failure, Success, Try}
+
```
Comment (Bromel777) [inline src/test/scala/org/encryfoundation/prismlang/integration/CostEstimatorSpec.scala]: empty string

**Pull request merged**